src/pool/srv_pool_map.c

```diff
@@ -372,27 +380,26 @@ ds_pool_map_tgts_update(struct pool_map *map, struct pool_target_id_list *tgts,
 		rc = pool_map_find_target(map, tgts->pti_ids[i].pti_id,
 					  &target);
 		if (rc <= 0) {
-			D_ERROR("Got request to change nonexistent target %u"
-				" in map %p\n",
-				tgts->pti_ids[i].pti_id, map);
+			D_ERROR(DF_MAP ": Got request to change nonexistent target %u\n",
+				DP_MAP(pool_uuid, map), tgts->pti_ids[i].pti_id);
 			return -DER_NONEXIST;
 		}
 
 		dom = pool_map_find_dom_by_rank(map, target->ta_comp.co_rank);
 		if (dom == NULL) {
-			D_ERROR("Got request to change nonexistent rank %u"
-				" in map %p\n",
-				target->ta_comp.co_rank, map);
+			D_ERROR(DF_MAP ": Got request to change nonexistent rank %u\n",
+				DP_MAP(pool_uuid, map), target->ta_comp.co_rank);
 			return -DER_NONEXIST;
 		}
 
-		rc = update_one_tgt(map, target, opc, &version, print_changes);
+		rc = update_one_tgt(pool_uuid, map, target, opc, &version, print_changes);
 		if (rc < 0)
 			return rc;
 
 		/* if the target status does not need to change */
```

Comment:
not sure but maybe the comment is intended to say this?
```suggestion
		/* if the rank status does not need to change */
```

Reply:
Changed to "rank".